test/test_pipeline.py

```diff
@@ -547,3 +548,45 @@ def run(self, **kwargs):
 
     output = pipeline.run(query="How old is John?")
     assert output["output"] == "question"
+
+
+@pytest.mark.elasticsearch
+@pytest.mark.parametrize(
+        "retriever,document_store",
+        [
+            ("embedding", "faiss"),
+            ("embedding", "milvus"),
+            ("embedding", "elasticsearch"),
+        ],
+        indirect=True,
+)
+def test_document_search_pipeline(retriever, document_store):
+    documents = [
+        {"text": "Sample text for document-1", "meta": {"source": "wiki1"}},
+        {"text": "Sample text for document-2", "meta": {"source": "wiki2"}},
+        {"text": "Sample text for document-3", "meta": {"source": "wiki3"}},
+        {"text": "Sample text for document-4", "meta": {"source": "wiki4"}},
+        {"text": "Sample text for document-5", "meta": {"source": "wiki5"}},
+    ]
+
+    document_store.write_documents(documents)
+    document_store.update_embeddings(retriever)
+
+    docs_id: list = []
+    for index, doc in enumerate(document_store.get_all_documents(return_embedding=False)):
```

Comment:
I believe this for loop is not really needed. We can define the ids above together with the rest of the document content `{"id"="a","text": "Sample text for document-1", "meta": {"source": "wiki1"}}`

Reply:
That totally slipped my mind. Done.